modules/generator/processor/servicegraphs/config.go

```diff
@@ -27,6 +27,9 @@ type Config struct {
 	// If client and server spans have the same attribute, behaviour is undetermined
 	// (either value could get used)
 	Dimensions []string `yaml:"dimensions"`
+
+	// If enabled X-SampleRatio attribute will be used for metric calculation
+	EnableSpanMultiplier bool `yaml:"enable_span_multiplier"`
```

Comment:
If there isn't a semantic convention for this attribute, we could use a string, that if defined, will be used as key to get the information.

Reply:
Yea we can have that